packages/twenty-front/src/modules/auth/sign-in-up/hooks/useSignInUp.tsx

```diff
@@ -118,31 +116,6 @@ export const useSignInUp = (form: UseFormReturn<Form>) => {
     ],
   );
 
-  useScopedHotkeys(
```

Comment:
Before the fix this was also overwritten by the Modal default hotkeyScope, but the behaviour was ensured by the handleKeyDow logic in SignInUpForm.

With the fix this works too while the logic is slightly different for the SignInUpStep.Password step, making them concurrent. We are replacing handleKeyDown logic with this one (adapting it with the desired logic, the one that has been executed until then, hence the move to SignInUpForm) as we want to favorise hotkeyScope.